checker/src/org/checkerframework/checker/initialization/InitializationVisitor.java

```diff
@@ -181,6 +184,19 @@ protected boolean checkContract(
                             return true;
                         }
                     }
+                } else if (fa.getReceiver() instanceof LocalVariable) {
+                    // TODO: what is the best way to handle other types of
+                    // receivers?
+                    Element elem = ((LocalVariable) fa.getReceiver()).getElement();
+                    AnnotatedTypeMirror recvType = atypeFactory.getAnnotatedType(elem);
```

Comment:
Instead of getting the type from the factory, is it sufficient to get the CFValue from the store?